docs/mddocs/Quickstart/llama_cpp_quickstart.md

```diff
@@ -316,3 +316,9 @@ Also, you can use `ONEAPI_DEVICE_SELECTOR=level_zero:[gpu_id]` to select device
 If you run the llama.cpp program on Windows and find that your program crashes or outputs abnormally when accepting Chinese prompts, you can open `Region->Administrative->Change System locale..`, check `Beta: Use Unicode UTF-8 for worldwide language support` option and then restart your computer.
 
 For detailed instructions on how to do this, see [this issue](https://github.com/intel-analytics/ipex-llm/issues/10989#issuecomment-2105600469).
+
+#### Sycl7.dll not found error
+If you meet `System Error: sycl7.dll not found`, please check if you are using a conda environment and in the right oneapi env, or if you executed `source /opt/intel/oneapi/setvars.sh` on Linux
```

Comment:
If you meet `System Error: sycl7.dll not found` on Windows or you meet similar error on Linux, please check
1. if you have installed conda and if you are in the right conda environment which has pip installed oneapi dependencies on Windows,
2. if you have executed source /opt/intel/oneapi/setvars.sh on Linux